app/components/Account/AccountPortfolioList.jsx

```diff
@@ -33,6 +33,7 @@ import {Tooltip, Icon as AntIcon} from "bitshares-ui-style-guide";
 import Translate from "react-translate-component";
 import AssetName from "../Utility/AssetName";
 import TranslateWithLinks from "../Utility/TranslateWithLinks";
+import _ from "lodash-es";
```

Comment:
Naming this to something better would probably make future debugging easier

Reply:
I agree. I made the change.